enterprise/app/jobs/account_sla_job.rb

```diff
@@ -0,0 +1,9 @@
+class AccountSlaJob < ApplicationJob
```

Comment:
lets namespace in under account./ sla  make the naming little bit more verbose 

Account::SlaResolutionSchedulerJob ?

Reply:
done